KONDUTKODEKSO.md

```diff
@@ -0,0 +1,25 @@
+# Kondutkodekso
+
+Ni estas kognaj.  
+Esti kogna estas esti limhava.  
+En nia limigo ni faras elektojn, kiuj estas malsaĝaj aŭ estas misaj.
+
+Se ni faras malsaĝajn elektojn pro nia limigeco,  
+ni ne rajtas juĝi aliaj pro la sama kialo.
+
+Do, ni ne povas juĝi,  
+tial ni pardonas.
+
+Tiu ĉi projekte kaj ĝiaj rezultaĵoj estas intencata kiel:  
+loko de lernado,  
+loko de kompreno,  
+loko de instruado,  
+loko de komunigo,  
+loko, kie kreantoj kreas la ilojn por helpi al aliaj kreantoj krei komplikajn aĵojn elegante.
+
+Fartu bone, kreanto. Fartu bone kaj kreu.
```

Comment:
In the English original "Creator" is capitalized here. As I assume that "creator" here is a mere categorization of the reader, not a name, nor a title or god/deity term, I've not capitalized "kreanto" here.